website/sql/schema.sql

```diff
@@ -54,6 +54,34 @@ CREATE TABLE `GameUser` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
+-- Table structure for table `Pairing`
+--
+
+DROP TABLE IF EXISTS `Pairing`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Pairing` (
```

Comment:
Discord discussion settled on GameTask. Not sure on ID fields, going with gametaskID for the moment.